### DISCUSSION TITLE
What we have to do as a member of GitHub Campus expert

### DISCUSSION THREAD

**Discussion by sarmadhamdani02** [community/community#56576] [GitHub Education]
### Hi everyone,

Hi everyone !👋
So, I want to apply for GitHub Campus Expert. The real reason behind this is to improve my networking and increase my knowledge in the field. But really don't have any idea what I have to do as a member of society. Do I have to organize some events?  Can someone please guide me? 🙂

**Comment by Ntshangase** [comment]
Hey @sarmadhamdani02 

- I'm also applying and in my understanding this program is meant for people who are part of a community that is already active and doing something. GitHub wants to come in to help you grow your community if you feel like there is something that they can help with.
- if you want to increase your networking and increase your knowledge in the filed, you can do that by building with the people around you, having weekly meeting to share what you have learned and assist each other would be a good place to start.
- for you to be able to apply for the campus expert program you need to have the  [GitHub student developer pack](https://education.github.com/discount_requests/application)

**Reply by sarmadhamdani02** [reply]
Hi @Ntshangase !
First of all, I am really grateful for the reply. You cleared most of the doubts but I still have the following confusion,

On, the website of Github Campus Expert (https://education.github.com/experts), it is written that, 
"If your application was successful, you’ll be invited to submit a 5-minute video introducing yourself and your **_community_**."
What do you think which community are we talking about? Do we have to need a group of people ready to join society as members and partners of ours? 🤔 If Yes, What if we are not having any such group of people?

**Reply by Ntshangase** [reply]
the 5-minute video is you talking about the group that you are apart of, and what you do as a group, what you would like GitHub to help you with, all that stuff.

If you don't have such people the best thing will be to start building such  a group and then you can apply for campus expert.
